src/protocols/secure_channel/CASESession.h

```diff
@@ -105,6 +104,7 @@ class DLL_EXPORT CASESession : public Messaging::ExchangeDelegate, public Pairin
      * @param peerAddress                   Address of peer with which to establish a session.
      * @param operationalCredentialSet      CHIP Certificate Set used to store the chain root of trust an validate peer node
      *                                      certificates
+     * @param opCredSetIndex                Index value used to choose the chain root of trust for establisbing a session
```

Comment:
The comment here doesn't really express how to determine this index.

Reply:
Updated.